database/seeds/PostStatusesTableSeeder.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class PostStatusesTableSeeder extends Seeder
+{
+    /**
+     * Seed the application's database.
+     *
+     * @return void
+     */
+    public function run()
+    {
+
+        App\PostStatus::firstOrCreate(['status' => \App\PostStatus::STATUS_PUBLISHED]);
```

Comment:
Move `\App\PostStatus` to `uses` block. It's not necessary to add full namespace everywhere. Keeping your code short increase readability.
The same for ohter files in PR

Reply:
Fixed.